src/ui/display_nbgl.c

```diff
@@ -116,6 +140,14 @@ static void start_transaction_callback(bool confirm) {
     }
 }
 
+static void start_transaction_callback_inverted(bool confirm) {
```

Comment:
A bit strange to have a function with this name. I suppose this comes from slightly 'abusing' `nbgl_useCaseChoice` in `ui_display_warning_generic`.

Perhaps cleaner to rename the callback to `display_warning_generic_callback` and move it right before  `ui_display_warning_generic`.